src/components/SimpleSelect.js

```diff
@@ -56,7 +56,7 @@ class SimpleSelect extends React.Component {
   _handleKeyDown = (e) => {
     if (keycode(e) === 'esc') {
       e.preventDefault();
-      this.props.onScrimClick();
+      this.props.onScrimClick(e);
```

Comment:
I don't think `onScrimClick` should know about the event. You should instead call `stopPropagation` in the click handler on line 105.